CHANGELOG.rst

```diff
@@ -6,11 +6,26 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
 
 .. _master-release:
 
-[Unreleased 0.13.1aX] - `master`_
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+[Unreleased 0.13.2aX] - `master`_
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note:: This version is not yet released and is under active development.
 
+Added
+-----
+
+Changed
+-------
+
+Fixed
+-----
+
+Removed
+-------
+
+[0.13.1] - 2019-04-15
```

Comment:
```suggestion
[0.13.1] - 2019-04-16
```
Looks like this made it into the release, but not a major issue

Reply:
ha, that's cause I opened the other PR yesterday, whoops